us_en/quests/roles_and_profiles.md

```diff
@@ -276,34 +274,45 @@ To get started creating your node definitions, open your `site.pp` manifest:
 
     vim /etc/puppetlabs/code/environments/production/manifests/site.pp
 
-Create a node definition block for your application server nodes. Use the
-`/^pasture-app/` regular expression as the title of your node definition and
-include the `role::pasture_app` role class.
+Create a node definition blocks for your application and database nodes.
+
+Remember, a node definition block can take a regular expression as its title.
+You want any node whose host name begins with `pasture-app` to be classified
+with the `role::pasture_app` role, so create a node definition block with the
+title `/^pasture-app/` and `include` the `role::pasture_app` role for matching
+nodes. Similarly, any node whose name matches `pasture-db` should be classified
+with the `role::pasture_db` role, so add a node definition with the title
+`/^pasture-db/` that includes this role.
+
+Now that you've implemented the roles and profiles pattern, remove the previous
+node definitions for the `pasture-app.puppet.vm` and `pasture-db.puppet.vm`
+nodes.
+
+When you're done, the section of your `site.pp` after the initial comments
+should look like the following.
 
 ```puppet
+node default {
+  # This is where you can declare classes for all nodes.
+  # Example:
+  #   class { 'my_class': }
+}
+
 node /^pasture-app/ {
   include role::pasture_app
 }
```

Comment:
Should the node definition for `^pasture-db` also be included here?

Reply:
@greglarkin If you look at the full changes for the file, you'll see that it is. The way GH displays the changes just puts some deleted lines in the midst of the site.pp file code block.

Reply:
Great, thank you @kjhenner